practice/src/javascripts/main.js

```diff
@@ -0,0 +1,31 @@
+import ProductList from './products/productList';
+import ProductTemplate from './templates/productTemplate';
+import ProductService from './services/productService';
+
+window.addEventListener('DOMContentLoaded', () => {
+  const service = new ProductService();
+  const template = new ProductTemplate();
+  const productList = new ProductList(service, template);
+  productList.renderProducts();
+});
+
+// Identify the links in the sidebar and add/remove the 'active' class when clicked
+document.querySelector('.nav-list').addEventListener('click', function (event) {
```

Comment:
@JennyDoJD should we need split into new function?

Reply:
Yes. I created a new function in another JS file